README.md

```diff
@@ -243,6 +243,20 @@ OPTIONS:
       Goroutines to process parallel uploads to backend. (default: 100)
       [$BAZEL_REMOTE_NUM_UPLOADERS]
 
+   --grpc_proxy.url value The base URL to use for a grpc proxy backend, e.g.
+      localhost:9090 or example.com:7070.
+      [$BAZEL_REMOTE_GRPC_PROXY_URL]
```

Comment:
I think we should probably provide a way to use TLS when talking to the proxy backend and not using mTLS. It looks like this PR uses TLS when authenticating with the proxy backend via mTLS (which makes sense), otherwise TLS is not used at all when talking to the proxy backend?